sqlglot/expressions.py

```diff
@@ -2109,6 +2110,10 @@ class Extract(Func):
     arg_types = {"this": True, "expression": True}
 
 
+class DatePart(Func):
```

Comment:
this will automatically pick up the token DATE_PART as this func, no need to make a token